sdk/search/azure-search-documents/azure/search/documents/_internal/_search_client.py

```diff
@@ -100,6 +101,15 @@ def close(self):
         """
         return self._client.close()
 
+    def get_search_indexing_buffered_sender(self, **kwargs):
+        # type: (dict) -> SearchIndexingBufferedSender
+        """Return a SearchIndexingBufferedSender instance
+
+        :rtype: ~azure.search.documents.SearchIndexingBufferedSender
+
+        """
+        return SearchIndexingBufferedSender(self._endpoint, self._index_name, self._credential, **kwargs)
```

Comment:
Is it possible to share the pipeline?  That's a big performance win for us in .NET but I'm not sure it translates across languages.

Reply:
This is a good question. Yes we can. But usually we don't do it by default. And we don't encourage Python users to use in this way.